core/templates/pages/admin-page/activities-tab/admin-prod-mode-activities-tab.directive.html

```diff
@@ -1,3 +1,9 @@
-<md-card class="oppia-page-card oppia-long-text" style="max-width: 700px">
+<md-card class="oppia-page-card oppia-long-text oppia-long-text-extra">
 The 'Activities' tab is not available in the production environment.
 </md-card>
+
+<style>
+  .oppia-long-text-extra {
```

Comment:
I see this used in a lot of places. Should we move this to oppia.css?

Reply:
Also, the names oppia-long-text and oppia-long-text-extra together is confusing.